src/formatDate.js

```diff
@@ -1,14 +1,36 @@
 'use strict';
 
-/**
+/*
  * @param {string} date
  * @param {string[]} fromFormat
  * @param {string[]} toFormat
  *
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[3]);
+  const receivedDate = {};
+  const changedDate = [];
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    switch (fromFormat[i]) {
+      case 'YYYY':
+        receivedDate['YY'] = dateArr[i].slice(2);
+        break;
+      case 'YY':
+        receivedDate['YYYY'] = (dateArr[i] <= 20 ? 20 : 19) + dateArr[i];
+        break;
+      default:
+        break;
```

Comment:
The `default` case is empty, which means that 'MM' and 'DD' parts are not being handled. You need to store these values in the `receivedDate` object as well, to ensure that all parts of the date are available for reformatting.